pinc/gradual.inc

```diff
@@ -127,4 +127,70 @@ function maybe_output_new_proofer_message()
     }
 }
 
+function encourage_p1_to_p2($round)
+{
+    global $code_url, $pguser;
+
+    if($round != 'P1')
+        return;
+
+    // see if the user already has access to P3 or P2, if they do just
+    // encourage them to proof there instead
+    $userSettings =& Settings::get_Settings($pguser);
+    $p2_access = $userSettings->get_boolean("P2.access");
+    $p3_access = $userSettings->get_boolean("P3.access");
+    if($p2_access || $p3_access)
+    {
+        if($p3_access)
+            $round_target = "P3";
+        else
+            $round_target = "P2";
+
+        echo "<div class='callout'>";
+        echo "<div class='calloutheader'>";
+        echo _("Consider working on projects in $round_target!");
+        echo "</div>";
+
+        echo "<p>Please consider proofreading a project in $round_target rather than here in P1. With the large influx of new users, having eligible users working in <a href='$code_url/tools/proofers/round.php?round_id=$round_target'>$round_target</a> helps significantly.</p>";
```

Comment:
Are we not localizing as this is temporary?

Reply:
Hm ... may just have missed it, as the title line is marked up. I didn't notice it, either.

Reply:
The markup in the title line was, frankly, accidental. This is Super Temporary so I don't want to bother with proper gettext markup.

I am noodling on ways to abstract this out to be less temporary which will get proper markup.